mteb/models/bm25.py

```diff
@@ -0,0 +1,131 @@
+from __future__ import annotations
+
+import logging
+from functools import partial
+from typing import Any, List, Optional, Union
+
+import bm25s
+import Stemmer
```

Comment:
Maybe import them within the functions? Importing them at the top makes them required by default rather than optional packages.

Reply:
Good suggestion. Done